keyboards/crkbd/rev1/split_util.c

```diff
@@ -20,18 +20,22 @@
 #endif
 
 #ifndef SPLIT_USB_TIMEOUT
-#    define SPLIT_USB_TIMEOUT 2500
+#    define SPLIT_USB_TIMEOUT 2000
+#endif
+
+#ifndef SPLIT_USB_TIMEOUT_POLL
+#    define SPLIT_USB_TIMEOUT_POLL 10
 #endif
 
 volatile bool isLeftHand = true;
 
 bool waitForUsb(void) {
-    for (uint8_t i = 0; i < (SPLIT_USB_TIMEOUT / 100); i++) {
+    for (uint8_t i = 0; i < (SPLIT_USB_TIMEOUT / SPLIT_USB_TIMEOUT_POLL); i++) {
         // This will return true of a USB connection has been established
```

Comment:
Might as well fix this typo while we're here

```suggestion
        // This will return true if a USB connection has been established
```

Reply:
Done for the other files too